pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -576,3 +596,256 @@ def input(self, args, key):
     def apply(self):
         """ Apply our selections. """
         self.data.autopart.type = self._selected_scheme_value
+
+class MountDataRec(kickstart.MountData):
+    """ An artificial subclass also recording changes """
+    def __init__(self, *args, **kwargs):
+        super(MountDataRec, self).__init__(*args, **kwargs)
+        self.modified = False
+        self.orig_format = None
+
+class MountPointAssignSpoke(NormalTUISpoke):
+    """ Assing mount points to block devices """
+    category = SystemCategory
+
+    def __init__(self, data, storage, payload, instclass):
+        NormalTUISpoke.__init__(self, data, storage, payload, instclass)
+        self.title = N_("Assing mount points")
+        self._container = None
+        self._mds = None
+
+        self._gather_mount_data_info()
+
+    def _gather_mount_data_info(self):
+        self._mds = OrderedDict()
+
+        for device in (d for d in self.storage.devicetree.leaves if not d.protected and d.size != Size(0)):
+            if device.format.type != "none":
```

Comment:
Shouldn't there be a constant for this ?

Reply:
Yes, there should be a constant for this, but it should be defined in *Blivet*, not here. And I don't want to introduce a dependency on a new release of blivet just because of this.